sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisClient.cs

```diff
@@ -37,7 +36,7 @@ public class DocumentAnalysisClient
         /// <remarks>
         /// Both the <paramref name="endpoint"/> URI string and the <paramref name="credential"/> <c>string</c> key
         /// can be found in the Azure Portal.
-        /// For more information see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/README.md#authenticate-a-form-recognizer-client"> here</see>.
```

Comment:
Funnily enough, the `Authenticate a Form Recognizer Client` didn't exist in the first place.

Reply:
Noooooooo lol